extension/third_party_attribute_extension_test.go

```diff
@@ -0,0 +1,82 @@
+package extension
+
+import (
+	"fmt"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/getyoti/yoti-go-sdk/v2/attribute"
+	"gotest.tools/assert"
+)
+
+func createDefinitionByName(name string) attribute.AttributeDefinition {
+	return attribute.NewAttributeDefinition(name)
+}
+
+func ExampleThirdPartyAttributeExtension() {
+	attributeDefinition := attribute.NewAttributeDefinition("some_value")
+
+	now, err := time.Parse("2006-01-02T15:04:05.999Z", "2019-10-30T12:10:09.458Z")
```

Comment:
Bit pedantic, but I don't think we should call this variable `now`, as it's not now anymore (unless we change it to store the value of `time.Now()`)